### DISCUSSION TITLE
Any extra config to run on Android 12?

### DISCUSSION THREAD

**Discussion by JaumeBoschAnglada** [yasirkula/UnityNativeGallery#267] [Q&A]
Hi,

i'm having troubles to run the plugion on an android 12 phone. I'm getting this error "Parameter "imagePath" is null or empty".
On android9 it works well.

Is there any extra andorixmanifest configuration to work with newer andorids?

**Comment by yasirkula** [comment]
Hi! May I see your code?

**Reply by JaumeBoschAnglada** [reply]
Hi,

It's strange.

If i call like this:

```
        NativeGallery.GetImageFromGallery((callback) =>
        {
            Debug.Log("CALLBACK" + callback);
        }
```
It doesn't work, the callback is empty.


BUT, If i call like this. It works (but it doesn't let me pick more than one image)

```
        NativeGallery.GetImagesFromGallery((callback) =>
        {
            Debug.Log("CALLBACK" + callback[0]);
        });
```

Thanks for your reply.

**Reply by yasirkula** [reply]
ACTION_PICK is used for GetImageFromGallery while ACTION_GET_CONTENT is used for GetImagesFromGallery. Why the callback isn't invoked for ACTION_PICK on your device, I wish I knew.. I wish I knew.. 😞 I'm so sick of Android ecosystem.

**Reply by yasirkula** [reply]
Can you spot any helpful logs in logcat?
